content/applications/marketing/events/revenues_report.rst

```diff
@@ -0,0 +1,161 @@
+===============
+Revenues report
+===============
+
+The Odoo *Events* application provides the ability to create custom reports, based on event-related
+data and analytics. These reports can either be focused on the *Attendees* or the *Revenues*. Both
+reporting pages offer their own unique array of metrics and options.
+
+The following documentation focuses solely on the reporting options related to event *Revenues*.
+
+Revenues reporting page
+=======================
+
+To access the *Attendees* reporting page in the Odoo *Events* application, navigate to
+:menuselection:`Events app --> Reporting --> Revenues`.
+
+.. image:: revenues_report/default-view.png
+   :align: center
+   :alt: The default view of the Revenues reporting page in the Odoo Events application.
+
+By default, the :guilabel:`Revenues` reporting page appears as a graph (a :icon:`fa-line-chart`
+:guilabel:`(line chart)` with :icon:`fa-database` :guilabel:`(stacked)` data). The default filters,
+:guilabel:`Non-free tickets` and :guilabel:`Event Start Date: (current year)`, are present in the
+:guilabel:`Search...` bar.
+
+.. tip::
+   To learn more about the various graph views (and graph view options), refer to the documentation
+   on :ref:`Graph views <reporting/using-graph>`.
+
+The :guilabel:`Attendees` reporting page can also be viewed as a :ref:`pivot table
+<reporting/views/pivot>`, by clicking the :icon:`oi-view-pivot` :guilabel:`(pivot table)` icon in
+the upper-right corner.
```

Comment:
```suggestion
The :guilabel:`Revenues` reporting page can also be viewed as a :ref:`pivot table
<reporting/views/pivot>`, by clicking the :icon:`oi-view-pivot` :guilabel:`(pivot table)` icon in
the upper-right corner.
```
I would move this above the tip block.

Reply:
While I fully appreciate this suggestion, I feel like, since this portion is discussing the pivot table view...it seems a bit "out of place" to put it above a graph-related admonition. So, taking that into consideration, I think I am going to leave this placement as-is...but, if I get a similar note down the line, I'll be sure to implement the suggestion (and give YOU 100% credit for suggesting it first! 👍 )